its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -181,22 +185,30 @@ public static String replaceLtsVersion(String version) {
   }
 
   public static void reset(Orchestrator orchestrator) {
-    orchestrator.getServer().newHttpCall("api/orchestrator/reset")
-      .setMethod(HttpMethod.POST)
+    // We add one day to ensure that today's entries are deleted.
+    Instant instant = Instant.now().plus(1, ChronoUnit.DAYS);
+
+    // The expected format is yyyy-MM-dd.
+    String currentDateTime = DateTimeFormatter.ISO_LOCAL_DATE
+      .withZone(ZoneId.of("UTC"))
+      .format(instant);
+
+    LOG.info("TEST SETUP: deleting projects analyzed before: " + currentDateTime);
```

Comment:
Do we need to log this? I'd remove the `currentDateTime` from the log, because it's in UTC. It runs against local SQ instance so there's no harm done.
I would just log that the all projects are deleted from local SQ (as it was in one of the intermediate commits)

Reply:
I think this helps with debugging since it shows the used format. The API is very restrictive in this regard and displaying the value can help in case of problems.